src/styles/blocks/scroll-up.scss

```diff
@@ -0,0 +1,25 @@
+.scroll-up {
+  position: relative;
+  &__img {
+    height: 25px;
+    width: 25px;
+    bottom: 195px;
+    right: 10px;
+    background-image: url(../images/icons/Scroll.svg);
+    background-repeat: no-repeat;
+    position: fixed;
+    opacity: .5;
+    text-decoration: none;
+    color: #000;
+    transition: .3s;
```

Comment:
the same, always add transition property. By default, transition property is all, and it's hard for the browser to control all properties.